Cargo.toml

```diff
@@ -19,14 +19,14 @@ exclude       = [
 
 [dependencies]
 log    = "0.3.1"
-nix    = { git = "https://github.com/carllerche/nix-rust", rev = "c4257f8a76" }
+nix    = "0.6.0"
 libc   = "0.2.4"
 slab   = "0.1.0"
 time   = "0.1.33"
 bytes  = "0.3.0"
 winapi = "0.2.1"
 miow   = "0.1.1"
-net2   = { version = "0.2.19", default-features = false }
+net2   = { version = ">=0.2.19,<=0.2.23", default-features = false }
```

Comment:
Ah yeah if the single quotes are changed to double quotes it'll parse successfully and Cargo will ignore it basically. We can add back in a few explicit sections (e.g. `target.${target_name}.dependencies`) for necessary platforms. All that being said though the case for older compatibility is pretty thin, so it may not be worth it.

In any case though I would not recommend merging this PR until this dependency ceiling is lifted as it's very likely to wreak havoc in the large ecosystem that depends on mio